### DISCUSSION TITLE
没有搞懂那个后端

### DISCUSSION THREAD

**Discussion by kitkw** [metatube-community/jellyfin-plugin-metatube#29] [General]
8080占用了改7070
docker run -d -p 7070:8080 -v $PWD/config:/config --name javtube javtube/javtube-server:latest -dsn /config/javtube.db
什么也没搜到
![1](https://user-images.githubusercontent.com/31963066/178123873-28b0b881-0a08-45c8-b062-9c789dc37820.png)

**Comment by kitkw** [comment]
搜到了原来自己设置错误。关帖

**Reply by Animism001** [reply]
你好，怎么设置的？上面的URL填上就好了吗

**Reply by adamlin30** [reply]
@Animism001 填入http://localhost:xxxx/即可，xxxx是你的port。

**Reply by wakloij66778** [reply]
咋设置哦 我8080后端配置好了 但是啥也刮不到。。。

**Reply by wakloij66778** [reply]
![image](https://github.com/metatube-community/jellyfin-plugin-metatube/assets/77519797/d5d57790-59f1-4e40-999d-d9c784a6ab7a)

**Reply by wakloij66778** [reply]
![image](https://github.com/metatube-community/jellyfin-plugin-metatube/assets/77519797/50650750-99d2-4d43-a196-277d34e8a5cb)
![image](https://github.com/metatube-community/jellyfin-plugin-metatube/assets/77519797/a299e181-b805-413d-9f4f-33185aadf5b7)

**Comment by chimiansw** [comment]
我也是http的就刮削不出东西，后来通过群晖的反代成https才可以，但是需要域名和ssl证书

**Reply by poko11** [reply]
我连不上本机IP。http://localhost:7070/不行，192.X.X.X:7070也不行
